src/Sql/Sql/help/Copy-AzSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -0,0 +1,338 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/powershell/module/az.sql/copy-azsqldatabaselongtermretentionbackup
+schema: 2.0.0
+---
+
+# Copy-AzSqlDatabaseLongTermRetentionBackup
+
+## SYNOPSIS
+Copies a long term retention backup to a target database.  
+
+## SYNTAX
+
+### CopyBackupDefault (Default)
+```
+Copy-AzSqlDatabaseLongTermRetentionBackup [-Location] <String> [-ServerName] <String> [-DatabaseName] <String>
+ [-BackupName] <String> [-ResourceGroupName <String>] -TargetDatabaseName <String>
+ [-TargetServerFullyQualifiedDomainName <String>] [-TargetServerName <String>] -TargetSubscriptionId <String>
+ -TargetResourceGroupName <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### CopyBackupByResourceId
+```
+Copy-AzSqlDatabaseLongTermRetentionBackup [-ResourceId] <String> -TargetDatabaseName <String>
+ [-TargetServerFullyQualifiedDomainName <String>] [-TargetServerName <String>] -TargetSubscriptionId <String>
+ -TargetResourceGroupName <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### CopyBackupByInputObject
+```
+Copy-AzSqlDatabaseLongTermRetentionBackup [-InputObject] <AzureSqlDatabaseLongTermRetentionBackupModel>
+ -TargetDatabaseName <String> [-TargetServerFullyQualifiedDomainName <String>] [-TargetServerName <String>]
+ -TargetSubscriptionId <String> -TargetResourceGroupName <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Copies a long term retention backup to a target database.  Target database can be in different cluster or environment from backup source.  
```

Comment:
Rephrase:
Copies a long term retention backup to a target server.  Target database can be in different cluster from backup source.  

Note: Please do not mention this can be used for cross-environment copy (that is special feature and shouldn't be mentioned here )